src/main/java/com/xpanxion/java/assignments/student2/Worker2.java

```diff
@@ -15,9 +18,13 @@ public void ex1() {
         depNames.put(1,"Electronics");
         depNames.put(2,"Food");
```

Comment:
Well.. I was looking to replace `depNames` with something that is not hardcoded.  Essentially, we need to convert the List of department names returned by  `DataAccess` to a Map.  https://howtodoinjava.com/java8/collect-stream-to-map/